Kitodo/src/main/java/org/kitodo/config/enums/ParameterCore.java

```diff
@@ -610,9 +610,19 @@
     /*
      * Elasticsearch properties
      */
-
     ELASTICSEARCH_BATCH(new Parameter<>("elasticsearch.batch", 500)),
-    ELASTICSEARCH_INDEXLIMIT(new Parameter<>("elasticsearch.indexLimit", 5000));
+    ELASTICSEARCH_INDEXLIMIT(new Parameter<>("elasticsearch.indexLimit", 5000)),
+
+    /*
+     * Security properties
+     */
+
+    /*
+     * Secret is used to encrypt or decrypt sensitive data e.g. LDAP manager passwords which are stored in the database in encrypted form.
+     * Once the secret is set, it should not be changed since encrypted data can no longer be decrypted.
+     */
+    SECURITY_SECRET(new Parameter<>("security.secret", ""));
```

Comment:
1. Please add this new parameter to the `kitodo_config.properties` files as this parameter is needed if you use a ldap server. 
2. Add your comment even to this parameter description in the files as this description is important too
3. Maybe rename this parameter to a more meaning full parameter name as the current name "security.secret" could mean a lot but this parameter is the secret for securing the used ldap manager password.

Reply:
For 3. I still need some input: I named the parameter more openly in order to use the secret with other encryption as well. 
Is this a case that does not occur, or should a secret be added for each case.